plugin-maven/src/main/java/com/diffplug/spotless/maven/java/ImportOrder.java

```diff
@@ -34,17 +36,43 @@ public class ImportOrder implements FormatterStepFactory {
 	@Parameter
 	private boolean wildcardsLast = false;
 
+	/**
+	 * Whether imports should be sorted based on semantics (i.e. sorted by package,
+	 * class and then static member). This considers <code>treatAsPackage</code> and
+	 * <code>treatAsClass</code>, and assumes default upper and lower case
+	 * conventions otherwise. When turned off, imports are sorted purely
+	 * lexicographically.
+	 */
+	@Parameter
+	private boolean semanticSort = true;
```

Comment:
We should not change the default behavior for existing users of the `importOrder` step.